src/commands/createAndSendCommand.ts

```diff
@@ -61,6 +57,26 @@ export const createAndSendTxCommand = {
     return yargs.options(createTxOptions);
   },
   handler: async (argv: CreateAndSendArgs) => {
+    if (!argv["params"]) {
```

Comment:
Why not use the `demandOption` instead ?

Reply:
it's not understood by Typescript it seems :/